### DISCUSSION TITLE
硕士论文封面

### DISCUSSION THREAD

**Discussion by ssevenss** [note286/xduts#27] [Q&A] [answered]
我想问下，这个硕士论文封面是黑体一号吗？ 为什么感觉跟word版本的封面字不一样呢？
![image](https://user-images.githubusercontent.com/103255322/209472077-7a148f10-6cea-4f0b-8f7d-40a6078f3e69.png)

**Comment by note286** [comment, marked as answer]
请自行根据手册更换中文字体配置。

**Reply by ssevenss** [reply]
好的~谢谢啦~

**Comment by pkeke** [comment]
请问一下，这个硕论的封面字体在哪修改啊

**Reply by note286** [reply]
请阅读PDF文档修改中文字体和英文字体配置。

如果你希望使用中易字体，可以参考如下示例：
```latex
\documentclass{xdupgthesis}
\xdusetup{style/cjk-font=win}
\begin{document}
\end{document}
```

**Comment by pkeke** [comment]
是我理解错了吗，我看你这个已经是1号字体了啊
![4](https://user-images.githubusercontent.com/44956389/211131998-b4e38631-9f9c-4466-90f8-e53b7472c165.png)

**Reply by note286** [reply]
请你清晰地描述你的问题，例如你认为哪一页那几个字的字号不正确。

**Reply by note286** [reply]
另外，你的问题与当前主题明显不一致，请单独在讨论区新开一个交流。

**Reply by pkeke** [reply]
我是看到上面的同学的提问说硕士论文封面可能不是1号字体

**Reply by pkeke** [reply]
但您的这个默认配置封面应该就是1号字体

**Reply by note286** [reply]
对此，我给你的答复是，没有问题。

另外，请区分字体和字号，题主的疑惑是Microsoft Word和LaTeX展示出来的glyph（字形）不同，原因我也给出了，是不同字体配置导致的，你可以自行切换字体配置。

再具体点说，xduts默认的中文字体是Fandol系列，Microsoft Word模板默认的字体是中易系列，学位办对此是无要求的，你想用方正、adobe、华文、中易或者其他的都随意。

**Reply by pkeke** [reply]
好的明白了，谢谢答复！
